hadoop-reader/src/main/java/uk/gov/gchq/palisade/service/data/hadoop/HadoopConfiguration.java

```diff
@@ -41,7 +41,7 @@
      * @throws IOException ioException
      */
     @Bean
-    @ConditionalOnProperty(prefix = "data", name = "implementation", havingValue = "hadoop")
+    @ConditionalOnProperty(prefix = "data", name = "implementation", havingValue = "hadoop", matchIfMissing = true)
```

Comment:
```suggestion
    @ConditionalOnProperty(prefix = "data", name = "implementation", havingValue = "hadoop")
```
I think you want the default reader if this config key is not set